common/cassandra/cassandraCluster.go

```diff
@@ -85,6 +106,8 @@ func NewCassandraCluster(cfg config.Cassandra) (*gocql.ClusterConfig, error) {
 	}
 	if cfg.ConnectTimeout > 0 {
 		cluster.ConnectTimeout = cfg.ConnectTimeout
+	} else {
+		cluster.ConnectTimeout = time.Minute
```

Comment:
why 1 minute default on connect timeout? no strong feelings just curious.

Reply:
yeah it turns out this was a terrible idea and was causing us to take up to 6 minutes to establish a session. I've since reverted this change.